### DISCUSSION TITLE
Is this post still util?

### DISCUSSION THREAD

**Discussion by enzzoperez** [strapi/strapi#6325] [Legacy Topics]
Hello!, I found this post but idk if still works: 

[https://strapi.io/blog/admin-permissions](url)

I need to create a thing like `admin` (already created by default) -  `editor` (only to add register in a collection)

Thanks!

**Comment by derrickmehaffy** [comment]
If you are referring to the admin, then please see: https://strapi.io/documentation/3.0.0-beta.x/getting-started/troubleshooting.html#user-can-t-login-to-the-admin-panel

**Reply by enzzoperez** [reply]
yes, thank you for response, that plugin will be very util for me
